scripts/generate-changelog.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Generate a changelog in markdown format from commits between two tags
+
+generate_changelog() {
+  #                    -c 'versionsort.suffix=v' is used to make "v1.2.3" be lesser version then "2.0.0"
+  local git_tags=$(git -c 'versionsort.suffix=v' tag --list --sort=-v:refname) # Get a list of all tags in reverse order
+  local tags_array=($git_tags) # Make the tags an array
+  local latest_tag="${tags_array[0]}"
```

Comment:
But if we are creating the release, then the changelog should go from the last tag to the latest commit, because the "GitHub Release" step needs the changelog.

So
[latest commit] 123
123
[latest tag] 456
456
[previous tag] 789

The changelog should have 123, 123. But right now it will have 456, 456.

Reply:
At the moment the release workflow in Bitrise creates the tag as [a separate step](https://github.com/salemove/android-sdk-widgets/blob/1920683727d499a9d1038ba42d4c235271ecc2ce/bitrise.yml#L267):
1. creates the git tag with the specified release version
2. generates the changelog
3. creates GitHub release with specified tag and changelog

But if needed I can make the changelog to generate from the last tag to the latest commit as you suggested. It is very simple change:
From this: https://github.com/salemove/android-sdk-widgets/blob/8a605aa5d0aae6230f41848f1e2abc30737e4edb/scripts/generate-changelog.sh#L13
To this: `local commits=$(git log "$latest_tag"..HEAD --no-merges --pretty=format:"%H")`